client/allocrunner/taskrunner/stats_hook.go

```diff
@@ -99,11 +99,14 @@ func (h *statsHook) collectResourceUsageStats(handle interfaces.DriverStats, sto
 					return
 				}
 
-				//XXX This is a net/rpc specific error
-				// We do not log when the plugin is shutdown as this is simply a
-				// race between the stopCollection channel being closed and calling
-				// Stats on the handle.
-				if !strings.Contains(err.Error(), "connection is shut down") {
+				// We do not log when the plugin is shutdown as this is either:
+				// - A race between the stopCollection channel being closed and
+				//	 calling Stats on the handle.
```

Comment:
Hm I kept the log there since it pre-dated me but I can't think of the race? The Shutdown method on the hook is always called in the graceful shutdown case before the driver exits so we should be good. Deleting that